tools/webdriver/webdriver/bidi/undefined.py

```diff
@@ -0,0 +1,6 @@
+class Undefined:
```

Comment:
But singletons have runtime checks. Why would a constant not be sufficient?

Reply:
I wasn't saying "don't make a constant", just that it's more normal to have something where trying to construct it will return the existing instance (unless you do something very involved) vs something which tries to stop you running the constructor.

(also in the context of running these tests, complaining about a single `if` statement as if it might be a noticable perf problem seems a bit silly).

Anyway I don't think it needs to block.

Reply:
Done.